.github/workflows/docker.build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         with:
           commit-message: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
           title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
-          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
+          branch: deploy-to-staging
```

Comment:
I think these need to be unique still, else we'll create conflicts in the `wbaas-deploy` repo when people do not instantly merge these PRs. Maybe we can find something else that is unique, but doesn't contain colons (which is why this broke). Maybe we can use `GITHUB_RUN_ID` instead which is a unique identifier for each workflow run: https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
that is a great idea. thanks